### ISSUE TITLE
Feature Requested

### ISSUE THREAD

**Issue by harisnaeemofficial** [popcorn-time-ru/popcorn-ru#50]
Hi, I want to suggest a feature whenever you have time to update this repo, 

Can you add that whenever a new user tries to search new IMDb id it should get the IMDb and TMDB data from API and get the torrent magnet link from YTS/torrent galaxy and any other popular torrent site and shows the result in the output console? 

LIKE: "https://movies-api.ml/show/tt1190634" 

This link shows the output {"message":"Not Found","code":404}

it should search the data from the IMDB/TMDB API and SCRAP/GET the torrent file from YTS/TORRENT-GALAXY and put that data immediately into the output screen and in the database (A new Movie/Show Added)

**Comment by ivan1986** [comment]
And where you think i need get this data?
We allready scan tmdb and yts and torrent galaxy, if we find torrent and info about film - it in database

**Comment by harisnaeemofficial** [comment]
No, you are scrapping the data by date and time with the cronjob and it scrapes overall movies on some specific page.

I am saying that if the movie is not in the database like I don't have this series "[tt1190634]" in my database and I want to get data immediately from spiders can you do this update?
like search tt1190634 id when user wants to search new movie/show

if (data is in database) {
show data from database
}
else if (if data is not in database){

(Search data from Spiders...)

data scraping...
Showing scrapped result...
image...                                   //img link
info...                                      //information of the movie
magnet link...                            // magnet links
else
{ 
data not found
}

I don't want to wait for spiders to search data any day and put data in the database because script scrapping cant scrape data immediately according to the ID "tt1190634"

**Comment by ottar-eten-yod** [comment]
I mean, I suppose it is possible to "search" but it will require some heavy code modification to get it to work. Will take a look.
